apple/Sources/FerrostarMapLibreUI/Views/Overylays/LandscapeNavigationOverlayView.swift

```diff
@@ -34,7 +35,10 @@ struct LandscapeNavigationOverlayView: View, CustomizableNavigatingInnerGridView
         onZoomOut: @escaping () -> Void = {},
         showCentering: Bool = false,
         onCenter: @escaping () -> Void = {},
-        onTapExit: (() -> Void)? = nil
+        onTapExit: (() -> Void)? = nil,
+        @ViewBuilder currentRoadNameViewBuilder: (String?) -> AnyView = { name in
+            AnyView(CurrentRoadNameView(currentRoadName: name, theme: DefaultRoadNameViewTheme()))
+        }
```

Comment:
This is probably fine as it's an internal view. No need to convert to view modifier.